src/providers/SignatureProvider.tsx

```diff
@@ -26,6 +28,7 @@ const createSignatureProviderState = () => {
     const [linkedin, setLinkedin] = useState<InputElementOptions>(getDefaultInputElementOptions("linkedin"));
     const [phone, setPhone] = useState<InputElementOptions>(getDefaultInputElementOptions("phone"));
     const [isDarkMode, setIsDarkMode] = useLocalStorageState(localStorageKeys.isDarkMode, false);
+    const [color, setColor] = useState<string>();
```

Comment:
We shouldn't need to have a dedicated useState for the color since we're adding it to the complex state object